kusto/kcsb.go

```diff
@@ -28,9 +34,15 @@ type ConnectionStringBuilder struct {
 	RedirectURL                      string
 	DefaultAuth                      bool
 	ClientOptions                    *azcore.ClientOptions
+	ApplicationForTracing            string
+	UserForTracing                   string
+	versionForTracing                string
```

Comment:
This is not a KCSB capability

Reply:
WDYM? I'll move the logic to another class, but we still need getters/setters for these in KCSB